### DISCUSSION TITLE
事实证明，只要你的vpn节点够多，那么这个绳网就能成功快速访问

### DISCUSSION THREAD

**Discussion by yingguan-Cloud** [share121/inter-knot#2836] [其他]
今天正常访问绳网，结果一直获取用户信息失败，获取文章失败，但当我换了40多个节点后，好了<img src="https://bbs-static.miyoushe.com/static/2024/06/28/5e73914d0c2348d5cb376a00ab82aad7_180874175435333649.png" alt="松雀-我吗" width="75" >

**Comment by YuFeiQAQ** [comment]
用的是steam++，这玩意能加速github

**Reply by AnonCN** [reply]
我给我的steam++，它说这是假的（悲）

**Comment by keyuanya** [comment]
你那边还得开梯子吗？我这里家里宽带和数据网络直连都没问题...🤔
![20240806_014050](https://github.com/user-attachments/assets/5a994acd-0480-4d1d-aafe-57a2f4050a5f)

**Reply by apiaoapiao** [reply]
流量少的话就可以

**Reply by aomengxin** [reply]
科拉莉可爱捏

**Comment by Loxic123** [comment]
用steam++会连不上，但是裸连就非常稳定<img src="https://bbs-static.miyoushe.com/static/2024/09/10/a858892127395d0d8f8cacf42479cacc_3718580379398331753.png" alt="貊泽 无语" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/09/10/a858892127395d0d8f8cacf42479cacc_3718580379398331753.png" alt="貊泽 无语" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/09/10/a858892127395d0d8f8cacf42479cacc_3718580379398331753.png" alt="貊泽 无语" width="75" >

**Reply by manyshoes** [reply]
steam++是这样的，很难评，有时候本来能裸连，开了之后连不上，再关掉更连不上。难说<img src="https://img-static.mihoyo.com/communityweb/upload/7bd889870907d2d96673d09afb53cf63.png" alt="三月七_哭" width="75" >
